contributors/guide/issue-triage.md

```diff
@@ -4,57 +4,233 @@ weight: 10
+- We skip issues that are closed
+- We also skip cherrypicks, because we consider that the code change was reviewed in the original PR
+- We ensure participation from the entire SIG and support company diversity. 
+- We use this opportunity to mark ["help needed", "good first issue"](#help-wantedgood-first-issues).
+
+### Triage Guide by cluster-lifecycle
+has developed a [triaging page](/sig-cluster-lifecycle/grooming.md) detailing their process, including the [Milestones](#planning-milestones) stage. Here is a [March 2020 presentation](https://www.youtube.com/watch?v=Q07_PfkNjlw) delivered to the SIG chairs and leads group on their process.
 
-These guidelines serves as a primary document for triaging incoming issues to
-Kubernetes. SIGs and projects are encouraged to either use these guidelines, or
-use this as a starting point if necessary. For example, if your SIG has specific
-triaging needs, extend these guidelines.
-**Note:** These guidelines only applies to the Kubernetes repository. Its usage
-for other GitHub repositories related to Kubernetes is TBD.
+## Step One: Review Newly Created Open Issues
+Kubernetes issues are listed [here](https://github.com/kubernetes/kubernetes/issues). New, untriaged issues come without labels attached. SIG leads should identify at least one SIG member to serve as a first point of contact for new issues.
 
-## Using the bot
+Labels are the primary tools for triaging. [Here's a comprehensive label list](https://github.com/kubernetes/kubernetes/labels).
 
-Most people can leave comments and open issues. They don't have the ability to
-set labels, change milestones and close other people's issues. For that we use
-a bot to manage labelling and triaging. The bot has a set of
-[commands and permissions](https://go.k8s.io/bot-commands)
-and this document will cover the basic ones.
+### Conducting Searches
+GitHub allows you to filter out types of issues and pull requests, which helps you discover items in need of triaging. This table includes some predetermined searches for convenience:
 
-## Determine if it's a support request
+|  Search | What it sorts  |
+|---|---|
+| [created-asc](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc)  | untriaged issues by age |
+|  [needs-sig](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-sig) | issues that need to be assigned to a SIG  |
+| [`is:open is:issue`](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue)   | Newest incoming issues  |
+| [comments-desc](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acomments-desc)   | busiest untriaged issues, sorted by # of comments  |
+| [comments-asc](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acomments-asc)   | Issues that need more attention, based on # of comments  |
```

Comment:
nit, keep the description consistent in this table with sentences starting with captial/small letter e.g. "issues that need to be assigned" (#2) vs "Issues that need more attention" (#5)

Reply:
Good catch